### PR TITLE
Add managementpartner sdk

### PR DESCRIPTION
**Pull request by jeffrey-ACE**
This SDK will be used to manage ManagementPartner 

PR for swaggers can be found at below:
https://github.com/Azure/azure-rest-api-specs/pull/2285
https://github.com/Azure/azure-rest-api-specs/pull/2351

### CODE REVIEW

File: azure-mgmt-managementpartner/setup.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'License :: OSI Approved :: MIT License',
+    ],
+    zip_safe=False,
+    packages=find_packages(exclude=["tests"]),
+    install_requires=[
+        'msrestazure~=0.4.11',
```
Comment (lmazuel) [inline azure-mgmt-managementpartner/setup.py]: `'msrestazure>=0.4.20,<2.0.0',`

File: azure-mgmt-managementpartner/setup.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    author='Microsoft Corporation',
+    author_email='azpysdkhelp@microsoft.com',
+    url='https://github.com/Azure/azure-sdk-for-python',
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
```
Comment (lmazuel) [inline azure-mgmt-managementpartner/setup.py]: We don't support 3.3 anymore

**Comment by lmazuel** [comment]
Could you share your plans? When API goes public as preview? GA? CLI? etc. so I can tell what should be the release story.
Thanks!

**Comment by jeffrey-ACE** [comment]
@lmazuel  This service is intended for partners.  The current target is make service and PowerShell/CLI live on Feb 28th.  We will work with few selected partners (15-25) and they will start using these PowerShell/CLI APIs.

**Comment by jeffrey-ACE** [comment]
@lmazuel thank you for merge this PR. How to publish the SDK to nuget?

**Pull request merged**